docs/bigquery/snippets.py

```diff
@@ -1037,6 +1103,46 @@ def test_load_table_from_uri_truncate(client, to_delete):
     # [END bigquery_load_table_gcs_json_truncate]
 
 
+def test_load_table_from_uri_parquet_truncate(client, to_delete):
+    dataset_id = 'load_table_dataset_{}'.format(_millis())
+    dataset = bigquery.Dataset(client.dataset(dataset_id))
+    client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    job_config = bigquery.LoadJobConfig()
+    job_config.schema = [
+        bigquery.SchemaField('name', 'STRING'),
+        bigquery.SchemaField('post_abbr', 'STRING')
+    ]
+    table_ref = dataset.table('us_states')
+    body = six.StringIO('Washington,WA')
```

Comment:
Nit: Per https://github.com/GoogleCloudPlatform/google-cloud-python/issues/4753 we should use BytesIO instead so that users who bump into this in our snippets files do the right thing when they have non-ASCII characters.

Reply:
Updated all `StringIO()` to `BytesIO()` in b3dd926